pkg/reflector/reflector.go

```
@@ -280,6 +294,50 @@ func ReflectorFromConfig(config ReflectorConfig) (*Reflector, error) {
 	}, nil
 }
 
+func emitMetricFromFile() error {
+	path := "/var/spool/ctlstore/metrics.json"
+	if _, err := os.Stat(path); err != nil {
+		if os.IsNotExist(err) {
+			return nil
+		}
+	}
```

you'll still want to return the err if its not NotExists.